### ISSUE TITLE
403 error when attempting to install Razer Chroma patch on 0.72 +804

### ISSUE THREAD

**Issue by Abcmsaj** [antonpup/Aurora#2002]
### Expected Behaviour
Should click install and update to the latest Installed version of Chroma in Aurora

### Actual Behaviour
Error message displayed seen here:

https://imgur.com/pCveUPq.jpg

### Frequency
Every time

### Aurora Version:
beta 0.7.2 +804 as I read in this post that it would fix the flickering issue I was seeing in OW

https://github.com/antonpup/Aurora/issues/1960

[2020-05-09 20.02.22.log](https://github.com/antonpup/Aurora/files/4604323/2020-05-09.20.02.22.log)

**Comment by diogotr7** [comment]
Razer took down this link. We're trying to figure out what the best way to handle this is.

**Comment by Abcmsaj** [comment]
Cool, thank you. I tried it on the latest stable release and the download worked? But then I ended up with #1960

**Comment by coggy9** [comment]
I managed to find a copy of what Aurora used to grab from Razer. [Here](https://mega.nz/file/c8YR3IgS#I90VEBxv0vJbNY1CXTY34aT8KsYll9duMme0ak3-HOk) is a link. Digitally signed by Razer, it's safe. Just run as admin then reopen Aurora. You might have to restart your PC if Aurora fails to connect. If a developer doesn't want links, I'll edit my comment.

**Comment by Abcmsaj** [comment]
![image](https://user-images.githubusercontent.com/5143023/81503556-2639bd00-92dc-11ea-9f05-b0c8260feae5.png)
You're a hero - thanks @coggy9 for the temp fix.

Was gonna suggest "is there a way these SDKs could be hosted elsewhere without legal ramifications from Razer" but I assume the answer's no...

**Comment by coreyshuman** [comment]
I can confirm the download link shared by @coggy9 does fix Aurora Chroma support. **However**, it will cause your Razer devices to exhibit the rainbowing effect (tested in Razer Chroma Emulator). It will also break the "Chroma Connect" feature of the Razer Synapse software.

@MoshiMoshi0 I'd be happy to help with `RazerSdkWrapper.dll` development if you're looking for assistance.

**Comment by MoshiMoshi0** [comment]
@coreyshuman If you know assembly you can help with reverse engineering the encryption they introduced in 3.12.0+. I have not yet had a chance to properly analyze the disassembly.

**Comment by coreyshuman** [comment]
@MoshiMoshi0 definitely, that sounds like a fun challenge. I had started to review the disassembly of the `ChromaEmulatorService.exe` since it looks like that one interfaces with the broadcast API and passes the data into a websocket for the emulator Electron app. I can continue down that route.

Would you be able to share the wrapper source or any docs/notes on the pre-encryption broadcaster to give me a head start understanding the underlying protocol?

**Comment by MoshiMoshi0** [comment]
@coreyshuman Ok, cool. But please lets continue this conversation on the aurora discord since this is off topic to the issue
